__tests__/apis.test.js

```diff
@@ -66,6 +66,104 @@ describe("4. GET /api/reviews", () => {
         });
       });
   });
+  test("responds with reviews ordered by ascending date", () => {
+    return request(app)
+      .get("/api/reviews?order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeInstanceOf(Array);
+        expect(reviews).toHaveLength(13);
+        expect(reviews).toBeSortedBy("created_at", { ascending: true });
+        reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
```

Comment:
The reason we get you to use either `.toMatchObject` or `.toEqual(expect.objectContaining())` is that you are allowing for extensibility. This means that you can rely on your previous tests that already assert the presence of these properties - you only really need to assert the sort order here 👍
ON SAYING THIS - these extra assertions certainly don't hurt!